INIT.md

```diff
@@ -1,10 +1,45 @@
-## webpack init
-
-Through [yeoman](http://yeoman.io/), the `webpack-cli init` feature allows people to create scaffolds and generate new projects quickly. An npm dependency that scaffolds a `webpack.config.js` through `webpack-cli` is what we refer to as an **addon**.
-
-We ask several questions in the default generator to get you started.
-
----
+# webpack-cli init
+
+`webpack-cli init` is used to initialize `webpack` projects quickly by scaffolding configuration and installing modules required for the project as per user preferences.
+
+## Initial Setup
+Follow given steps to setup `webpack-cli init` by installing dependencies:
+1. Create `package.json` through npm
+
+	```shell
+    	$ npm init
+    ```
+
+2. Install `webpack` and `webpack-cli` as devDependencies
+
+	```shell
+    	$ npm install --save-dev webpack webpack-cli
+    ```
+
+3. Optional step : Install @webpack-cli/init package to add init addon
+
+	```shell
+    	$ npm install -D @webpack-cli/init
+    ```
+    
+4. Add following line under scripts in `package.json`
+
+	```json
+    	...
+        "scripts": {
+        	...
+            "project-init" : "webpack-cli init",
```

Comment:
dhruvdutt is right, there's no need of the npm script. If the user followed up to the second step, they can simply run the cli command through 

```bash 
npx webpack-cli init
```
If the package is installed locally, or

```bash
webpack-cli init
```
If the package is installed globally. I think we should make this distinction during all the steps, otherwise the users gets confused

Reply:
Does npx comes up with npm? as i think it is a package that needed to be installed using npm itself, increasing dependency. maybe?!

Reply:
Yes npx comes with npm out of the box! It has no deps and it won't be part of the package.json.

Reply:
Great! just discovered and is coming since npm 5.2!! 
So, should we also add up global setup @ematipico ?

Reply:
I would say so! So we'd cover all cases and we clear up everything!